opencog/rule-engine/Rule.cc

```diff
@@ -648,4 +652,4 @@ std::string oc_to_string(const RuleTypedSubstitutionMap& rules)
 	return oc_to_string(rules, "");
 }
 
-} // ~namespace opencog
+} // ~namespace opencog
```

Comment:
Now that's not enough new line, you file needs to end by one new line (not two, just one).